RoundCheckbox.js

```diff
@@ -27,19 +32,90 @@ export default class RoundCheckbox extends React.PureComponent {
     onValueChange: () => {},
   };
 
+  state = {
+    scaleAndOpacityOfCheckbox: null,
+  };
+
+  componentDidMount() {
+    this.setState({
+      scaleAndOpacityOfCheckbox: new Animated.Value(this.props.checked ? 1 : 0),
+    });
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.checked !== prevProps.checked) {
+      console.log(this.state.scaleAndOpacityOfCheckbox);
+      if (this.props.checked) {
+        Animated.timing(this.state.scaleAndOpacityOfCheckbox, {
```

Comment:
you can simplify this:

```
Animated.timing(this.state.scaleAndOpacityOfCheckbox, {
          toValue: this.props.checked ? 1 : 0,
          duration: 300,
          useNativeDriver: false,
        }).start();
```

I also believe useNativeDriver can be true

Reply:
I think it can be as well. I had it set to `false` for debugging, I'll update